### PR TITLE
Hot reload client SDK and openapi.yaml

### PR DESCRIPTION
**Pull request by anmho**
Run `make watch`. Making changes to backend will automatically `openapi.yaml` and client SDKs.

Created cli command that outputs `openapi.yaml` without starting the server.

Call `make openapi` to output `openapi.yaml` manually.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,12 @@
 
-
 deps:
 	@npm install -g orval
 
-.PHONY: openapi gen
-
```
Comment (anmho) [inline Makefile:7]: unnecessary commands

File: api/cmd/api/main.go [hunk 2]
```diff
@@ -91,12 +89,39 @@ func main() {
 	logger.Info("server listening", slog.Int("port", Port))
 	// Generate openapi.yaml after the server starts
 	go func() {
+		// Wait for server start
 		time.Sleep(time.Millisecond * 250)
+
```
Comment (anmho) [inline api/cmd/api/main.go:94]: Wait for server readiness check /ping before generating the new openapi.yaml and client SDK using orval.

File: api/.air.toml [hunk 3]
```diff
@@ -6,11 +6,9 @@ tmp_dir = "tmp"
   args_bin = []
   bin = "./bin/api"
   cmd = "make api"
-  delay = 1000
```
Comment (anmho) [inline api/.air.toml:9]: minimize delay

File: api/Makefile [hunk 4]
```diff
@@ -15,11 +17,13 @@ test:
 api:
 	@go build -o ./bin/api ./cmd/api
 
-gen:
```
Comment (anmho) [inline api/Makefile:18]: Not using buf anymore

File: api/cmd/api/main.go [hunk 5]
```diff
@@ -43,6 +42,56 @@ func pgConnString(config Config) string {
 		config.DbName)
 }
 
+func createOpenAPIAndClientSDK() {
```
Comment (anmho) [inline api/cmd/api/main.go]: Launched in a goroutine on startup that creates the updated openapi.yaml and orval sdk.

**Review by Jish2 — APPROVED** [review]

**Pull request merged**